sdcm/utils/issues_by_keyword/find_known_issue.py

```diff
@@ -0,0 +1,38 @@
+import logging
+from collections import defaultdict
+from pathlib import Path
+from typing import NamedTuple
+
+import yaml
+
+from sdcm.utils.metaclasses import Singleton
+
+LOGGER = logging.getLogger(__name__)
+
+
+class MapKeywords(NamedTuple):
+    keyword: str
+    issue: str
+
+
+# pylint: disable=too-few-public-methods
+class FindIssuePerBacktrace(metaclass=Singleton):
+
+    MAPPING_FILE = Path(__file__).parent / "issue_by_keyword.yaml"
+
+    def __init__(self):
+        with open(file=self.MAPPING_FILE, encoding="utf-8", mode="r") as map_file:
+            yaml_details = yaml.safe_load(map_file)
+
+        self.map = defaultdict(list)
+        for backtrace_type, values in yaml_details.items():
+            for one_pair in values:
+                self.map[backtrace_type].append(MapKeywords(keyword=one_pair["keyword"], issue=one_pair["issue"]))
+        LOGGER.debug("MapKeywords: %s", self.map)
+
+    def find_issue(self, backtrace_type: str, decoded_backtrace: str) -> str | None:
+        LOGGER.debug("Search for an issue for %s backtrace", backtrace_type)
+        for all_keywords in self.map.get(backtrace_type, []):
+            if all_keywords.keyword in decoded_backtrace:
```

Comment:
we might consider finding more then one issue per backtraces ? and make it known_issues ?

Reply:
Not that I am aware about such case

Reply:
It maybe few keywords for one issue.
If we open the issue by backtrace and there is the already opened an issue for the same/similar backtrace, the new opened issue is closed as duplicate. Even it is OSS and Enterprise
So I think it is not expected few issues for same backtrace

Reply:
I thought more of cases we have multiple issue within the same backtrace...
Don't know if it gonna happen a lot or not, but not that critical for now, if we'll need it, it would be easily expended